src/stan-users-guide/style-guide.Rmd

```diff
@@ -152,15 +156,27 @@ Because Stan ignores whitespace and the parser completes a statement
 as eagerly as possible (just as in C++), the previous program is
 equivalent to the following program.
 
-```
+```stan
 for (n in 1:N) {
   z[n] ~ normal(nu,1);
 }
 y[n] ~ normal(mu,1);
 ```
 
-Therefore, one should use braces in all instances.
+Therefore, one should prefer to use braces. The only exception is when
```

Comment:
We decided no exceptions.  The lines 171--179 are not exceptions to the brackets everywhere rule, so I don't see why they're being called out.

Reply:
OK, that's fine.  I think this crosses over more into refactoring than style.   I think there are a lot of things along those lines we could start recommending.  I added an issue to add a refactoring chapter to the user's guide.